src/types/index.ts

```diff
@@ -110,9 +110,9 @@ export interface IWallet {
 }
 
 export interface IHdWalletAccount {
-  publicKey: string;
-  secretKey: string;
-  address: string,
+  publicKey: string | Uint8Array;
+  secretKey: string | Uint8Array;
```

Comment:
I'm not sure if this is correct. We have two kinds of objects that holds the `publicKey` and `secretKey`. This is one of them and it's related to the data that is actually stored in the `LocalStorage`. So it can't be anything different than `string`.

The `Uint8Array` values are assigned to `IKeyPar`. I suggest to not mix this here.

Reply:
The Bitcoin adapter was casting `publicKey` & `sercetKey` `as any`. So they are saved as Buffers. 
If that is the case with `LocalStorage` then perhaps we should save them all as `strings` and not cast them

Reply:
So maybe the adapters should convert the keys to string as it will eventually happen when saving to/from the `LocalStorage`?